staging/src/k8s.io/kube-aggregator/hack/update-codegen.sh

```diff
@@ -60,7 +60,7 @@ ${CLIENTGEN} --clientset-name="clientset" ${INPUT_BASE} --input apiregistration/
 
 echo "Building lister-gen"
 listergen="${PWD}/lister-gen"
-go build -o "${listergen}" ${CODEGEN_PKG}/cmd/lister-gen
+go build ${GOFLAGS:-} -o "${listergen}" ${CODEGEN_PKG}/cmd/lister-gen
```

Comment:
this lgtm, but I don't know why we need to pass those flags anyway ?

I have noticed that some `sub-projects` call directy `code-gen/generate-groups.sh` and `code-gen/generate-internal-groups.sh` scripts. For example https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/metrics/hack/update-codegen.sh

However the scripts rely on `go install` to build binaries
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/code-generator/generate-groups.sh#L53, so I think that they don't respect `GOFLAGS`